### ISSUE TITLE
Formatting html documents out of the currently opened folder's scope

### ISSUE THREAD

**Issue by mciesin** [prettier/prettier-vscode#1117]
### Summary

Cannot format html documents out of the currently opened folder's scope.

### Steps To Reproduce:

1. Open a folder.
2. Open an html file out of the opened folder's scope.
3. Try to format the file.

### Expected result

File gets formatted.

### Actual result

`Extension 'Prettier - Code formatter' cannot format ...`

### Additional information

**VS Code Version**: 1.40.2

**Prettier Extension Version**: 3.11.0

**OS and version**: Windows 10

## Prettier Log Output

```
["INFO" - 10:23:54] Extension Name: "prettier-vscode".
["INFO" - 10:23:54] Extension Version: "3.11.0".
["INFO" - 10:23:55] Enabling prettier in workspace 'framelogic' for languages:
[
  "javascript",
  "mongo",
  "javascriptreact",
  "typescript",
  "typescriptreact",
  "json",
  "jsonc",
  "json5",
  "css",
  "postcss",
  "less",
  "scss",
  "graphql",
  "markdown",
  "mdx",
  "html",
  "vue",
  "yaml"
]
["INFO" - 10:23:55] Enabling prettier for range supported languages
[
  "javascript",
  "javascriptreact",
  "typescript",
  "typescriptreact",
  "json",
  "graphql"
]
```

**Comment by sanket-bhalerao** [comment]
i am facing same issue.

**Comment by sanket-bhalerao** [comment]
reverted back to v3.9.0 to get around the issue for time being.

**Comment by ntotten** [comment]
This should be fixed in 3.13+. Please reopen if you continue to see the issue.
